selene/browser.py

```diff
@@ -34,141 +34,183 @@
 
 
 def driver() -> WebDriver:
-    warnings.warn('selene.browser.driver is deprecated, '
-                  'use `from selene.support.shared import browser` import',
-                  DeprecationWarning)
+    warnings.warn(
+        'selene.browser.driver is deprecated, '
+        'use `from selene.support.shared import browser` import',
+        DeprecationWarning,
```

Comment:
comma on the last multiline argument, helps to refactor:
When you add new argument, you don't add comma, so there is no -1 line modified, only + N lines added in commit changes.